### DISCUSSION TITLE
Frame Drops on HDR/Dolby Vision Content on TCL TV with Android TV

### DISCUSSION THREAD

**Discussion by psychedelicu** [jellyfin/jellyfin-androidtv#3981] [General]
Hello,

I'm experiencing frequent frame drops (not stuttering) on HDR/Dolby Vision 4K content while using Jellyfin and Emby on my TCL C805 TV running Google TV (Android v12). This occurs during Direct Play and affects only HDR/Dolby Vision content—SDR and x264 content play without issues. The same problem also occurs on Netflix.

It seems the TV remains locked at 60Hz, causing frame drops when playing 23Hz content. Both "Enable Refresh Rate Switching" and "Match Video Resolution" settings do not appear to work correctly on this TV or Android TV platform in general.

Are there any known issues or fixes for frame rate switching on TCL TVs or Android TV?
This is severely affecting the viewing experience for HDR content. Any help would be appreciated.

Also, the playback info is missing on Jellyfin for Android app. 

Thank you.

**Comment by xrvo** [comment]
I am having the same issue with the Android TV app on Sony A80K.  I experience frequent dropped frames playing 4K HDR HEVC content.  This is in in Direct Play as well.  I also noticed that a horizontal band of sparse white pixels (sort of like analog static TV noise, but only with white pixels overlayed on the image) flashes across the screen for one frame every approx. 30 seconds of video.

This seems to have only started happening recently, although I can't yet confirm whether content that previously played without issue is also affected.

**Reply by JIVS** [reply]
Which codec(s) are you guys using?

**Reply by xrvo** [reply]
> Which codec(s) are you guys using?

Here are typical video codec details for a few files exhibiting this issue:
```
Title: 4K HEVC Dolby Vision Profile 8.1 (HDR10)
Codec: HEVC
AVC: No
Profile: Main 10
Level: 150
Resolution: 3840x1606
Aspect ratio: 2.40:1
Anamorphic: No
Interlaced: No
Framerate: 23.976025
Bitrate: 24997 kbps
Bit depth: 10 bit
Video range: HDR
Video range type: DOVIWithHDR10
DV title: Dolby Vision Profile 8.1 (HDR10)
DV version major: 1
DV version minor: 0
DV profile: 8
DV level: 6
DV rpu preset flag: 1
DV el preset flag: 0
DV bl preset flag: 1
DV bl signal compatibility id: 1
Color space: bt2020nc
Color transfer: smpte2084
Color primaries: bt2020
Pixel format: yuv420p10le
Ref frames: 1
```
